### ISSUE TITLE
[Debian 7 x64] [Build 1.1.1-7] omireg register provider failed

### ISSUE THREAD

**Issue by Wayenyin** [Microsoft/omi-script-provider#30]
**ENV:**
- Debian 7 x64

**Repos:**
1. Install omi version 1.3.0-105;
2. Install python script build:[ \\\\redmond\wsscfs\OSTCData\Builds\omiscriptprovider\develop\1.1.1-7](url) ;
3. Currently copy customized python provider to /opt/omi/lib, it will be fixed #22;
4. Switch to above directory: "_cd /opt/omi/lib_";
5. Register Python provider but failed: "_/opt/omi/bin/omireg --Python xyz_frog_"
    ```
    /opt/omi/lib
    root@non64-deb7-01  # /opt/omi/bin/omireg --Python xyz_frog
    Traceback (most recent call last):
       File "client.py", line 7, in <module>
          import omi
    ImportError: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by /usr/local/lib/python2.7/dist-packages/omi.so)
    /opt/omi/bin/omireg: MI_Module.schemaDecl is null
    ```
**Check libc.so.6 version:**
   Run command: "_ls -al /lib/x86_64-linux-gnu/|grep libc.so.6_"
   ```
   root@non64-deb7-01  # ls -al /lib/x86_64-linux-gnu/|grep libc.so.6
   lrwxrwxrwx  1 root root      12 Dec 30  2012 libc.so.6 -> libc-2.13.so
   ```

**Comment by alinbalutoiu** [comment]
Please check with the latest build (1.1.1-13 at the moment) to see if this issue still exists.

**Comment by Wayenyin** [comment]
This issue is fixed on Python script provider build 1.1.1-14.
